projects/sunbird-video-player/src/lib/components/video-player/video-player.component.ts

```diff
@@ -116,14 +105,16 @@ export class VideoPlayerComponent implements AfterViewInit, OnDestroy {
   }
 
   toggleForwardRewindButton() {
```

Comment:
call this method if it is mobile only

Reply:
this method is called in multiple places. Checking isMobile in every place will duplicate the code